Library/Formula/libdistance.rb

```diff
@@ -0,0 +1,12 @@
+class Libdistance < Formula
+  homepage "http://monkey.org/~jose/software/libdistance/"
+  url "http://monkey.org/~jose/software/libdistance/libdistance-0.2.2.tar.gz"
+  sha1 "d516e5f24ae55bcbbf93a4d14c6a53914e5b3a0a"
+
+  def install
+    # The build system is incredibly simple.
+    system "make"
+    include.install "distance.h"
+    lib.install "libdistance.a"
+  end
+end
```

Comment:
Can this test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!